docs-gen/content/docs/reference/configuration.md

```diff
@@ -86,13 +87,17 @@ The configured `serviceAccount` needs to have [workload identity](https://cloud.
   - `storage.objects.create` on the configured `pipelineBucket`
   - `storage.objects.get` on the configured `pipelineBucket`
   - `storage.objects.delete` on the configured `pipelineBucket`
-  - `cloudscheduler.jobs.create`
-  - `cloudscheduler.jobs.update`
-  - `cloudscheduler.jobs.delete`
+  - deprecated - `cloudscheduler.jobs.create`
+  - deprecated - `cloudscheduler.jobs.update`
+  - deprecated - `cloudscheduler.jobs.delete`
   - `projects.topics.publish` to the configured `runs` and `runIntentsTopic` topic
```

Comment:
this should also be deprecated

Reply:
true, didn't want to go too far since going to be all removed in next change. But have added this one for you